README.md

```diff
@@ -38,9 +38,10 @@ After completing installation you can add this card like any other to your dashb
 3. Click _+ New Card_ button
 4. Select `Custom: Meteoalarm Card`
 5. Select meteoalarm entity. If you don't see one make sure you have configured [Meteoalarm integration](https://www.home-assistant.io/integrations/meteoalarm/)
+
 ```yaml
-type: 'custom:meteoalarm-card'
-entity: 'binary_sensor.meteoalarm'
+type: "custom:meteoalarm-card"
+entity: "binary_sensor.meteoalarm"
```

Comment:
Actually YAML generated from adding cards by UI uses single quotes so it would be best to keep using them

![image](https://user-images.githubusercontent.com/23432278/118661423-6422c880-b7ef-11eb-930e-42231eaa2f73.png)

Reply:
Sorry, that was my Markdown linter, will change.